landoapi/api/revisions.py

```diff
@@ -26,6 +27,28 @@ def get(revision_id, api_key=None):
     return _format_revision(phab, revision, include_parents=True), 200
 
 
+def land(revision_id, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    phab = PhabricatorClient(api_key)
+    revision = phab.get_revision(id=revision_id)
+    revision = _format_revision(phab, revision, include_parents=True)
+    if not revision:
+        # We could not find a matching revision.
+        return problem(
+            404,
+            'Revision not found',
+            'The requested revision does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+
+    trans = TransplantClient()
+    id = trans.land(
```

Comment:
Can we make the TransplantClient#land method accept named arguments. Having to know the position of the arguments makes things harder to remember.